ngsderive/__main__.py

```diff
@@ -1,21 +1,19 @@
 #!/usr/bin/env python3
 
 import argparse
-import csv
 import logging
 import sys
 
-from ngsderive import utils
 from ngsderive.commands import (
-    readlen,
-    instrument,
-    strandedness,
     encoding,
-    junction_annotation,
     endedness,
+    instrument,
+    junction_annotation,
+    readlen,
+    strandedness,
 )
 
-logger = logging.getLogger("ngsderive")
+logger = logging.getLogger("")
```

Comment:
This should really be the following:

```suggestion
logger = logging.getLogger()
```

This gets the root logger with a log level of WARN.